mne/viz/_figure.py

```diff
@@ -0,0 +1,1854 @@
+        vline_text = ax_hscroll.text(
+            self.mne.first_time, 1.2, '', fontsize=10, ha='right', va='bottom',
+            **vline_kw)
+
+        # HELP BUTTON: initialize in the wrong spot...
+        ax_help = div.append_axes(position='left',
+                                  size=Fixed(help_width),
+                                  pad=Fixed(vscroll_dist))
+        # HELP BUTTON: ...move it down by changing its locator
+        loc = div.new_locator(nx=0, ny=0)
+        ax_help.set_axes_locator(loc)
+        # HELP BUTTON: make it a proper button
+        # XXX when matplotlib 3.0 is min version,
+        # XXX uncomment next line and remove self._post_init()
+        # self.mne.button_help = Button(ax_help, 'Help')
+        # PROJ BUTTON
+        ax_proj = None
+        if len(self.mne.projs) and not inst.proj:
+            proj_button_pos = [
+                1 - self._inch_to_rel(r_margin + scroll_width),  # left
+                self._inch_to_rel(b_margin, horiz=False),        # bottom
+                self._inch_to_rel(scroll_width),                 # width
+                self._inch_to_rel(scroll_width, horiz=False)     # height
+            ]
+            loc = div.new_locator(nx=4, ny=0)
+            ax_proj = self.add_axes(proj_button_pos)
+            ax_proj.set_axes_locator(loc)
+            # XXX when matplotlib 3.0 is min version,
+            # XXX uncomment next line and remove self._post_init()
+            # self.mne.button_proj = Button(ax_proj, 'Prj')
+
+        # INIT TRACES
+        self.mne.traces = ax_main.plot(
+            np.full((1, self.mne.n_channels), np.nan),
+            antialiased=True, linewidth=0.5)
+
+        # SAVE UI ELEMENT HANDLES
+        vars(self.mne).update(
+            ax_main=ax_main, ax_help=ax_help, ax_proj=ax_proj,
+            ax_hscroll=ax_hscroll, ax_vscroll=ax_vscroll,
+            vsel_patch=vsel_patch, hsel_patch=hsel_patch, vline=vline,
+            vline_hscroll=vline_hscroll, vline_text=vline_text,
+            fgcolor=fgcolor, bgcolor=bgcolor)
+
+    def _post_init(self):
+        """Make buttons (fix for older matplotlib (older than 3.0?) XXX)."""
+        from matplotlib.widgets import Button
+        # HELP BUTTON: make it a proper button
+        self.mne.button_help = Button(self.mne.ax_help, 'Help')
+        if len(self.mne.projs) and not self.mne.inst.proj:
+            self.mne.button_proj = Button(self.mne.ax_proj, 'Prj')
+
+    def _close(self, event):
+        """Handle close events (via keypress or window [x])."""
+        from matplotlib.pyplot import close
+        # write temporary info object back to instance (for bads, but don't
+        # persist changes to proj checkboxes)
+        self.mne.info['projs'] = self.mne.inst.info['projs']
+        self.mne.inst.info = self.mne.info
+        # Clean up child figures too
+        while len(self.mne.child_figs):
+            fig = self.mne.child_figs[-1]
```

Comment:
This looks like a bug and that it should be `self.mne.child_figs.pop(-1)` until looking deeper into the code. Can you add a comment here that the child figures all remove themselves from the parent figure lists in `_close` so need to `pop`?